src/Components/Publishing/Partner/PartnerInline.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import Events from "../../../Utils/Events"
+import track from "../../../Utils/track"
+import { pMedia } from '../../Helpers'
+import Icon from '../../Icon'
+import { IconPlus } from '../Icon/IconPlus'
+
+interface Props {
+  logo: string
+  url: string
+  tracking?: any
+  trackingData?: any
+}
+
+@track((props) => {
+  return props.trackingData ? props.trackingData : {}
+}, {
+  dispatch: data => Events.postEvent(data)
+})
+export class PartnerInline extends React.Component<Props, null> {
+  static defaultProps = {
+    tracking: {
+      trackEvent: x => x
+    }
+  }
+
+  constructor(props) {
+    super(props)
+    this.onPartnerClick = this.onPartnerClick.bind(this)
+  }
+
+  onPartnerClick(event) {
+    this.props.tracking.trackEvent({
```

Comment:
Looking at the share tracking now, what is the props.tracking object and where does it come from?

Reply:
Comes from the track decorator!